README.md

```diff
@@ -138,3 +158,71 @@ HTTP Response:
     "count": int
 }
 ```
+
+## Adding Alarm Metric Log
+
+POST "/metrics/alarm"
+
+Request Body:
+
+```
+{
+    "alarmInfo": [
```

Comment:
There is no batch POST, so you dont need an array, just have username startTime and endTime as root attributes to the resquest body